source/module_io/input.cpp

```diff
@@ -369,7 +369,7 @@ void Input::Default(void)
 
     exx_separate_loop = true;
     exx_hybrid_step = 100;
-    exx_mixing_beta = 0.0;
+    exx_mixing_beta = 1.0;
```

Comment:
Please modify `module_io/tests`(which failed on CI) corresponding to the change of default value. Thanks~